.travis.yml

```diff
@@ -1,8 +1,30 @@
 language: go
+
+go:
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
+  - "1.10.x"
+  - master
+
+matrix:
+  # Build OK if fails on unstable development versions of Go.
+  allow_failures:
+    - go: master
+  # Don't wait for tests to finish on allow_failures.
+  # Mark the build finished if tests pass on other versions of Go.
+  fast_finish: true
+
 go_import_path: firebase.google.com/go
+
 before_install:
-    - go get github.com/golang/lint/golint
+    - go get github.com/golang/lint/golint #Golint requires Go 1.6 or later.
+
+install:
+    # prio to go 1.8 package with only test go get returns : "no buildable Go source files ..."
```

Comment:
Lets change this as: `# Prior to golang 1.8, this can trigger an error for packages containing only tests.`